packages/manager/modules/pci/src/projects/project/baremetal/onboarding/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,16 @@
+{
+  "pci_projects_project_baremetal_onboarding_content1": "Vous n'avez pas encore créé d’instance.",
+  "pci_projects_project_baremetal_onboarding_content2": "Déployez des instances parmi une large gamme de modèles et profitez de la flexibilité du cloud pour croître selon vos besoins.",
+  "pci_projects_project_baremetal_onboarding_content3": "Dans notre catalogue d'instances, choisissez le modèle qui vous convient parmi la gamme Sandbox, dont les ressources sont partagées, la gamme Guaranteed Resources, aux performances constantes pour les applications exigeantes, la gamme GPU, idéale pour les calculs parallélisés, et la gamme IOPS, pour les bases de données ou le big data.",
```

Comment:
This looks like the previous description

Reply:
We are actually waiting for the content. Until we get that, it is the same description as instances :)

Reply:
The content has been updated. Thanks :)